cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CreateExecutableTask.java

```diff
@@ -163,7 +146,34 @@ public void execute(Project project) {
         notifyPlugins(project, target);
     }
 
-    private void notifyPlugins(Project project, Target target) {
+    private Path getExecutablePath(Project project, Target target) {
+        Path executablePath;
+        try {
+            executablePath = target.getExecutablePath(project.currentPackage()).toAbsolutePath().normalize();
+        } catch (IOException e) {
+            throw createLauncherException(e.getMessage());
+        }
+        return executablePath;
```

Comment:
```suggestion
        try {
            return target.getExecutablePath(project.currentPackage()).toAbsolutePath().normalize();
        } catch (IOException e) {
            throw createLauncherException(e.getMessage());
        }
```

Reply:
Done in [40229a4](https://github.com/ballerina-platform/ballerina-lang/pull/42183/commits/40229a40c6e8385fc2d56f004746b7a5494411f1) commit.